src/main/java/uk/gov/hmcts/divorce/document/content/templatecontent/TemplateContent.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.divorce.document.content.templatecontent;
+
+import uk.gov.hmcts.divorce.divorcecase.model.Applicant;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+
+import java.util.List;
+import java.util.Map;
+
+public interface TemplateContent {
+    List<String> getSupportedTemplates();
+
+    Map<String, Object> getTemplateContent(CaseData caseData, Long caseId, Applicant applicant);
```

Comment:
Didn't this need to be an ordered map?

Reply:
Maybe we should use https://docs.oracle.com/javase/6/docs/api/java/util/LinkedHashMap.html - not strictly ordered but it does maintain the iteration order

Reply:
Ah as in you think we should make the method signature force a return of a Map implementation that is ordered? If so then yeah will do